src/extensions/surveys/components/QuestionTypes.tsx

```diff
@@ -158,7 +158,14 @@ export function RatingQuestion({
                             className="rating-options-number"
                             style={{ gridTemplateColumns: `repeat(${scale - starting + 1}, minmax(0, 1fr))` }}
                         >
-                            {(question.scale === 5 ? fiveScaleNumbers : tenScaleNumbers).map((number, idx) => {
+                            {(question.scale === 5
```

Comment:
Maybe we should break this out into a variable. Its pretty confusing to read nested ternary statements here.

Reply:
done